.travis.yml

```diff
@@ -1,17 +1,12 @@
 language: c
-sudo: false
-services:
-  - docker
+sudo: required
```

Comment:
Sudo is usually not required for docker builds

Reply:
I've fixed this and will merge this PR when it has built.